topology_builder/pom.xml

```diff
@@ -33,18 +33,18 @@
     <dependency>
       <groupId>org.apache.storm</groupId>
       <artifactId>storm-core</artifactId>
-      <version>0.9.4</version>
```

Comment:
I'm cool with the changes, but you bumped Storm and Kafka versions here. This is a big change and deserves a minor pyleus release and thorough testing. I'm not against it, though. 

Storm 0.9.4 seems a pretty safe upgrade (http://storm.apache.org/2015/03/25/storm094-released.html), while Kafka 0.8.2.1 may be a bit more troublesome.

Asking @ecanzonieri or @patricklucas to weigh in.